docs/guides/src/main/operator/customizing-keycloak.adoc

```diff
@@ -7,26 +7,26 @@
 title="Using custom Keycloak images"
 summary="How to customize and optimize the Keycloak Container">
 
-== Keycloak custom image with Operator
+== Keycloak custom image with the Operator
 
-The Keycloak Custom Resource (CR) give you the possibility to specify a custom container image for the Keycloak server.
+With the Keycloak Custom Resource (CR), you can specify a custom container image for the Keycloak server.
 
 [NOTE]
 To ensure full compatibility of Operator and Operand,
 make sure that the version of Keycloak release used in the custom image is aligned with the version of the operator.
 
-=== Best Practice
+=== Custom image and build configuration
```

Comment:
I would like to suggest the users what we consider a best practice. Without this heading, I thing the empasis on building a custom image is not as strong as it was.